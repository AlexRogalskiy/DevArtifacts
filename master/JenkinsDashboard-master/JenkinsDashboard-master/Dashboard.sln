
Microsoft Visual Studio Solution File, Format Version 12.00
# Visual Studio 14
VisualStudioVersion = 14.0.24720.0
MinimumVisualStudioVersion = 10.0.40219.1
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Dashboard", "Dashboard\Dashboard.csproj", "{AC718715-251E-4415-83B5-9BCBC99BBCBC}"
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "JenkinsClient.Net", "JenkinsClient\JenkinsClient.Net.csproj", "{F23009A9-1782-41F0-8D6D-2B5BB1A17195}"
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "JenkinsClient.pcl", "JenkinsClient.pcl\JenkinsClient.pcl.csproj", "{9D735A55-BE6B-431C-B352-603186C49904}"
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Dashboard.Desktop", "Dashboard.Desktop\Dashboard.Desktop.csproj", "{2A9842FE-DE59-4253-B589-15DE052B0C15}"
EndProject
Global
	GlobalSection(SolutionConfigurationPlatforms) = preSolution
		Debug|Any CPU = Debug|Any CPU
		Debug|ARM = Debug|ARM
		Debug|x64 = Debug|x64
		Debug|x86 = Debug|x86
		Release|Any CPU = Release|Any CPU
		Release|ARM = Release|ARM
		Release|x64 = Release|x64
		Release|x86 = Release|x86
	EndGlobalSection
	GlobalSection(ProjectConfigurationPlatforms) = postSolution
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|Any CPU.ActiveCfg = Debug|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|Any CPU.Build.0 = Debug|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|Any CPU.Deploy.0 = Debug|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|ARM.ActiveCfg = Debug|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|ARM.Build.0 = Debug|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|ARM.Deploy.0 = Debug|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|x64.ActiveCfg = Debug|x64
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|x64.Build.0 = Debug|x64
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|x64.Deploy.0 = Debug|x64
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|x86.ActiveCfg = Debug|x86
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|x86.Build.0 = Debug|x86
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Debug|x86.Deploy.0 = Debug|x86
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|Any CPU.ActiveCfg = Release|x86
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|ARM.ActiveCfg = Release|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|ARM.Build.0 = Release|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|ARM.Deploy.0 = Release|ARM
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|x64.ActiveCfg = Release|x64
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|x64.Build.0 = Release|x64
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|x64.Deploy.0 = Release|x64
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|x86.ActiveCfg = Release|x86
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|x86.Build.0 = Release|x86
		{AC718715-251E-4415-83B5-9BCBC99BBCBC}.Release|x86.Deploy.0 = Release|x86
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Debug|ARM.ActiveCfg = Debug|ARM
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Debug|ARM.Build.0 = Debug|ARM
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Debug|x64.ActiveCfg = Debug|x64
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Debug|x64.Build.0 = Debug|x64
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Debug|x86.ActiveCfg = Debug|Any CPU
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Release|Any CPU.Build.0 = Release|Any CPU
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Release|ARM.ActiveCfg = Release|ARM
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Release|ARM.Build.0 = Release|ARM
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Release|x64.ActiveCfg = Release|x64
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Release|x64.Build.0 = Release|x64
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Release|x86.ActiveCfg = Release|x86
		{F23009A9-1782-41F0-8D6D-2B5BB1A17195}.Release|x86.Build.0 = Release|x86
		{9D735A55-BE6B-431C-B352-603186C49904}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Debug|ARM.ActiveCfg = Debug|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Debug|ARM.Build.0 = Debug|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Debug|x64.ActiveCfg = Debug|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Debug|x64.Build.0 = Debug|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Debug|x86.ActiveCfg = Debug|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Debug|x86.Build.0 = Debug|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Release|Any CPU.Build.0 = Release|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Release|ARM.ActiveCfg = Release|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Release|ARM.Build.0 = Release|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Release|x64.ActiveCfg = Release|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Release|x64.Build.0 = Release|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Release|x86.ActiveCfg = Release|Any CPU
		{9D735A55-BE6B-431C-B352-603186C49904}.Release|x86.Build.0 = Release|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Debug|ARM.ActiveCfg = Debug|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Debug|ARM.Build.0 = Debug|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Debug|x64.ActiveCfg = Debug|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Debug|x64.Build.0 = Debug|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Debug|x86.ActiveCfg = Debug|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Debug|x86.Build.0 = Debug|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Release|Any CPU.Build.0 = Release|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Release|ARM.ActiveCfg = Release|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Release|ARM.Build.0 = Release|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Release|x64.ActiveCfg = Release|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Release|x64.Build.0 = Release|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Release|x86.ActiveCfg = Release|Any CPU
		{2A9842FE-DE59-4253-B589-15DE052B0C15}.Release|x86.Build.0 = Release|Any CPU
	EndGlobalSection
	GlobalSection(SolutionProperties) = preSolution
		HideSolutionNode = FALSE
	EndGlobalSection
EndGlobal
