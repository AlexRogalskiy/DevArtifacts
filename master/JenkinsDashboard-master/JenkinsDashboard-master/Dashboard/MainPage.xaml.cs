using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Runtime.InteropServices.WindowsRuntime;
using Windows.Foundation;
using Windows.Foundation.Collections;
using Windows.UI.Xaml;
using Windows.UI.Xaml.Controls;
using Windows.UI.Xaml.Controls.Primitives;
using Windows.UI.Xaml.Data;
using Windows.UI.Xaml.Input;
using Windows.UI.Xaml.Media;
using Windows.UI.Xaml.Navigation;

// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409

namespace Dashboard
{
    /// <summary>
    /// An empty page that can be used on its own or navigated to within a Frame.
    /// </summary>
    public sealed partial class MainPage : Page
    {
        public MainPage()
        {
            this.InitializeComponent();
        }

        private async void button_Click(object sender, RoutedEventArgs e)
        {
            var serverInfo = new JenkinsClient.JenkinsServerInfo() { JenkinsServer = "" };

            var loader = new JenkinsClient.JenkinsDataLoader(serverInfo);
            var proj = await loader.GetProjectData("");

            if (proj.builds.Count > 0)
            {
                var bi = await loader.GetBuildInformation(proj.builds[0]); }


        }
    }
}
