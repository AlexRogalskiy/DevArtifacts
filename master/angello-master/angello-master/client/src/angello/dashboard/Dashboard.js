angular.module('Angello.Dashboard', ['Angello.Common']);