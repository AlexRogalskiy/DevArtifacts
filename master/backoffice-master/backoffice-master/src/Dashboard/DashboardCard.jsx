import React from 'react'
import PropTypes from 'prop-types'
import classNames from 'classnames'

import {
  Card,
  CardContent,
  Typography,
  withStyles,
  Avatar,
} from '@material-ui/core'

import DisabledIcon from '@material-ui/icons/Lock'

const styles = theme => ({
  root: {
    cursor: 'pointer',
  },
  content: {
    paddingBottom: `${theme.spacing.unit * 2}px !important`,
  },
  avatar: {
    float: 'left',
    marginRight: theme.spacing.unit * 2,
    backgroundColor: theme.palette.secondary.dark,
  },
  title: {
    marginBottom: 0,
  },
  disabled: {
    opacity: 0.75,
    pointerEvents: 'none',
    filter: 'grayscale(30%)',
  },
  icon: {
    float: 'right',
    opacity: 0.5,
  },
})

const DashboardCard = ({
  title,
  description,
  handleClick,
  icon: Icon,
  isDisabled,
  classes,
}) => {
  const rootClasses = classNames({
    [classes.root]: true,
    [classes.disabled]: isDisabled,
  })

  return (
    <Card onClick={handleClick} className={rootClasses}>
      <CardContent className={classes.content}>
        {isDisabled ? (
          <DisabledIcon className={classes.icon} />
        ) : null}

        {Icon ? (
          <Avatar className={classes.avatar}>
            <Icon />
          </Avatar>
        ) : null}

        <Typography variant="headline" className={classes.title} component="h2">
          {title}
        </Typography>

        {description ? (
          <Typography variant="body1">{description}</Typography>
        ) : null}
      </CardContent>
    </Card>
  )
}

DashboardCard.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string,
  handleClick: PropTypes.func.isRequired,
  icon: PropTypes.func,
  isDisabled: PropTypes.bool,
  classes: PropTypes.objectOf(PropTypes.string).isRequired,
}

DashboardCard.defaultProps = {
  description: null,
  icon: null,
  isDisabled: false,
}

export default withStyles(styles)(DashboardCard)
