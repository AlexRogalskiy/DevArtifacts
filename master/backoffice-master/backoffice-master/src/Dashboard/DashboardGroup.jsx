import React from 'react'
import PropTypes from 'prop-types'

import {
  Grid,
  Typography,
  withStyles,
} from '@material-ui/core'

import DashboardCard from './DashboardCard'

const styles = theme => ({
  headline: {
    marginTop: theme.spacing.unit * 3,
    marginBottom: theme.spacing.unit * 2,
  },
})

const DashboardGroup = ({
  title,
  cards,
  onClick,
  classes,
}) => (
  <div>
    <div className={classes.headline}>
      <Typography variant="headline">
        {title}
      </Typography>
    </div>

    <Grid container spacing={16}>

      {cards ? cards.map(card => (
        <Grid item xs={12} sm={4} key={`group-card-${card.id}`}>
          <DashboardCard
            title={card.title}
            description={card.description}
            handleClick={() => onClick(card.link)}
            icon={card.icon}
            isDisabled={card.isDisabled}
          />
        </Grid>
      )) : null}

    </Grid>
  </div>
)

DashboardGroup.propTypes = {
  title: PropTypes.string.isRequired,
  cards: PropTypes.arrayOf(PropTypes.object).isRequired,
  onClick: PropTypes.func.isRequired,
  classes: PropTypes.objectOf(PropTypes.string).isRequired,
}

export default withStyles(styles)(DashboardGroup)
