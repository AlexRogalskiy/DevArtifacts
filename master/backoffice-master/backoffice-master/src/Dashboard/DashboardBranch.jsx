import React from 'react'
import PropTypes from 'prop-types'
import { Typography, withStyles } from '@material-ui/core'

import DashboardGroup from './DashboardGroup'

const styles = theme => ({
  root: {
    flexGrow: 1,
    maxWidth: theme.spacing.unit * 140,
    marginLeft: 'auto',
    marginRight: 'auto',
  },
  headline: {
    marginTop: theme.spacing.unit * 3,
    marginBottom: theme.spacing.unit * 2,
  },
})

const DashboardBranch = ({ data, onClick, classes }) => (
  <div className={classes.root}>
    <div className={classes.headline}>
      <Typography variant="display2">{data.title}</Typography>
      <Typography variant="body1">{data.description}</Typography>
    </div>

    {data.groups ? data.groups.map(group => (
      <DashboardGroup
        key={`group-${group.id}`}
        onClick={onClick}
        {...group}
      />
      )) : null}
  </div>
)

DashboardBranch.propTypes = {
  data: PropTypes.objectOf(PropTypes.any).isRequired,
  onClick: PropTypes.func.isRequired,
  classes: PropTypes.objectOf(PropTypes.string).isRequired,
}

export default withStyles(styles)(DashboardBranch)
