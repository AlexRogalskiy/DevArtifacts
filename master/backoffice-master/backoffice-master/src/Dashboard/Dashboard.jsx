import React from 'react'
import PropTypes from 'prop-types'
import { withRouter } from 'react-router-dom'

import DashboardBranch from './DashboardBranch'

const withDashboard = Component => class extends React.Component {
  static propTypes = {
    history: PropTypes.objectOf(PropTypes.any).isRequired,
  }

  constructor() {
    super()

    this.handleClick = this.handleClick.bind(this)
  }

  handleClick(url) {
    this.props.history.push(url)
  }

  render() {
    return (
      <Component
        {...this.props}
        onClick={this.handleClick}
      />
    )
  }
}

const Dashboard = withDashboard(DashboardBranch)

export default withRouter(Dashboard)
