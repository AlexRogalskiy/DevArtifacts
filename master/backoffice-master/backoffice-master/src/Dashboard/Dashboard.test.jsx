import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import { mount } from 'enzyme'

import Dashboard from './'

it('renders correctly', () => {
  const tree = mount((
    <Router>
      <Dashboard
        data={{
          title: 'Backoffice',
          description: 'Visual Testing for Backoffice Framework',
          groups: [{
            title: 'Title',
            cards: [{
              title: 'title',
              description: 'desc',
            }],
          }],
        }}
      />
    </Router>
  ))

  expect(tree).toMatchSnapshot()
})
