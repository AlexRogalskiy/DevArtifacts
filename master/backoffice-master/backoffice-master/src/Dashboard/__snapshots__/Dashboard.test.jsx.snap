// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="DashboardBranch-root-1"
>
  <div
    className="DashboardBranch-headline-2"
  >
    <h1
      className="MuiTypography-root-3 MuiTypography-display2-6"
    >
      Backoffice
    </h1>
    <p
      className="MuiTypography-root-3 MuiTypography-body1-12"
    >
      Visual Testing for Backoffice Framework
    </p>
  </div>
  <div>
    <div
      className="DashboardGroup-headline-28"
    >
      <h1
        className="MuiTypography-root-3 MuiTypography-headline-8"
      >
        Title
      </h1>
    </div>
    <div
      className="MuiGrid-container-29 MuiGrid-spacing-xs-16-52"
    >
      <div
        className="MuiGrid-item-30 MuiGrid-grid-xs-12-69 MuiGrid-grid-sm-4-75"
      >
        <div
          className="MuiPaper-root-133 MuiPaper-elevation1-136 MuiPaper-rounded-134 MuiCard-root-132 DashboardCard-root-126"
          onClick={[Function]}
        >
          <div
            className="MuiCardContent-root-160 DashboardCard-content-127"
          >
            <h2
              className="MuiTypography-root-3 MuiTypography-headline-8 DashboardCard-title-129"
            >
              title
            </h2>
            <p
              className="MuiTypography-root-3 MuiTypography-body1-12"
            >
              desc
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
